src/core/constants.ts

```diff
@@ -181,6 +181,15 @@ export const SWA_CONFIG_FILENAME_LEGACY = "routes.json";
 export const CUSTOM_URL_SCHEME = "swa://";
 export const OVERRIDABLE_ERROR_CODES = [400, 401, 403, 404];
 
+export const TELEMETRY_MAC_ADDRESS_HASH_LENGTH = 32; //32 bytes
+export const MACHINE_ID_LENGTH = 16; //16 bytes
+
+export const TELEMETRY_LOGIN_EVENT = "login";
```

Comment:
nit: Just a thought, can we have all these as a single const like this
export const TELEMETRY_EVENTS = {
  Login: "login",
  Start: "start"
}
and while using we can use like TELEMETRY_EVENTS.Login etc..,
Feel free to change/leave it as you see fit